### PR TITLE
Documented the security options related to redirections

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: reference/configuration/security.rst [hunk 1]
```diff
@@ -318,10 +318,37 @@ request to the ``check_path`` URL.
...
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-* ``always_use_default_target_path`` (type: ``boolean``, default: ``false``)
-* ``default_target_path`` (type: ``string``, default: ``/``)
-* ``target_path_parameter`` (type: ``string``, default: ``_target_path``)
-* ``use_referer`` (type: ``boolean``, default: ``false``)
+always_use_default_target_path
+..............................
+
+**type**: ``boolean`` **default** ``false``
```
Comment (xabbuh) [inline reference/configuration/security.rst]: missing colon

File: reference/configuration/security.rst [hunk 2]
```diff
@@ -318,10 +318,37 @@ request to the ``check_path`` URL.
...
+
+If ``true``, users are always redirected to the default target path regardless
+of the previous URL that was stored in the session.
+
+default_target_path
+....................
+
+**type**: ``string``, **default**: ``/``
```
Comment (xabbuh) [inline reference/configuration/security.rst]: We do not have commas anywhere else.

File: reference/configuration/security.rst [hunk 3]
```diff
@@ -318,10 +318,37 @@ request to the ``check_path`` URL.
...
+
+The page users are redirect to when there is no previous page stored in the
+session (for example when the users access directly to the login page).
+
+target_path_parameter
+.....................
+
+**type**: ``string``, **default**: ``_target_path``
```
Comment (xabbuh) [inline reference/configuration/security.rst]: same here

File: reference/configuration/security.rst [hunk 4]
```diff
@@ -318,10 +318,37 @@ request to the ``check_path`` URL.
...
+
+If ``true``, users are always redirected to the default target path regardless
+of the previous URL that was stored in the session.
+
+default_target_path
+....................
+
+**type**: ``string``, **default**: ``/``
+
+The page users are redirect to when there is no previous page stored in the
```
Comment (xabbuh) [inline reference/configuration/security.rst]: [...] are redirected to [...]

File: reference/configuration/security.rst [hunk 5]
```diff
@@ -318,10 +318,37 @@ request to the ``check_path`` URL.
...
+session (for example when the users access directly to the login page).
```
Comment (xabbuh) [inline reference/configuration/security.rst]: [...] for example, when users browse the login form directly [...]

File: reference/configuration/security.rst [hunk 6]
```diff
@@ -318,10 +318,37 @@ request to the ``check_path`` URL.
...
+
+**type**: ``string``, **default**: ``_target_path``
+
+When using a login form, if you include an HTML element to set the target path,
+this option lets you change the name of the HTML element itself.
+
+use_referer
+...........
+
+**type**: ``boolean``, **default**: ``false``
```
Comment (xabbuh) [inline reference/configuration/security.rst]: the comma should be removed

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Review by HeahDude — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**